website/content/en/docs/Getting started/installation.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Installation"
+linkTitle: "Installation"
+weight: 2
+description: >
+  How to install AAD Pod Identity on your clusters.
+---
+
+## Quick Install
+
+To install/upgrade AAD Pod Identity on RBAC-enabled clusters:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.6.3/deploy/infra/deployment-rbac.yaml
+```
+
+<details>
+<summary>Result</summary>
+
+```
+serviceaccount/aad-pod-id-nmi-service-account created
+customresourcedefinition.apiextensions.k8s.io/azureassignedidentities.aadpodidentity.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/azureidentitybindings.aadpodidentity.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/azureidentities.aadpodidentity.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/azurepodidentityexceptions.aadpodidentity.k8s.io created
+clusterrole.rbac.authorization.k8s.io/aad-pod-id-nmi-role created
+clusterrolebinding.rbac.authorization.k8s.io/aad-pod-id-nmi-binding created
+daemonset.apps/nmi created
+serviceaccount/aad-pod-id-mic-service-account created
+clusterrole.rbac.authorization.k8s.io/aad-pod-id-mic-role created
+clusterrolebinding.rbac.authorization.k8s.io/aad-pod-id-mic-binding created
+deployment.apps/mic created
+```
+
+</details><br/>
+
+To install/upgrade aad-pod-identity on RBAC-disabled clusters:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.6.3/deploy/infra/deployment.yaml
+```
+
+<details>
+<summary>Result</summary>
+
+```
+customresourcedefinition.apiextensions.k8s.io/azureassignedidentities.aadpodidentity.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/azureidentitybindings.aadpodidentity.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/azureidentities.aadpodidentity.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/azurepodidentityexceptions.aadpodidentity.k8s.io created
+daemonset.apps/nmi created
+deployment.apps/mic created
+```
+
+</details><br/>
+
+For AKS clusters, you will have to allow MIC and AKS add-ons to access IMDS without being intercepted by NMI:
```

Comment:
Lets add a warning note here - `Failure to apply this yaml for AKS clusters will result in token failures for AKS addons using managed identity for auth`